client/src/components/Details.jsx

```diff
@@ -24,14 +24,14 @@ class Details extends React.Component {
     let readMore = null;
```

Comment:
On line 17 (not shown in this comment view) - you have "event" listed as input in to the `handleReadMoreClick` function but you don't seem to use it. Consider removing 'event'.